### DISCUSSION TITLE
Gravity sphere's base no longer rotates.

### DISCUSSION THREAD

**Discussion by inmanw** [FakeFishGames/Barotrauma#14902] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Only the sphere rotates now with seems pointless. I like using them as decorations and can't anymore due to them not being adjustable.

### Reproduction steps

1 Load up sub editor.
2 spawn gravity sphere.
3 adjust rotation.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the upcoming hotfix patch. Closing discussion.
